src/utils/calcCanvasNodeSizeAndImg.ts

```diff
@@ -44,7 +44,7 @@ const calcCanvasNodeSizeAndImg = async ({
   nodeType,
   extensionAPI,
 }: {
-  text: string;
+  text: string | number | Date;
```

Comment:
[nit] `text` has a distinct meaning throughout the codebase(s) as the string value of a roam block. I'd rather we:
- cast to string before inserting into this method
- calling it `value` or something like that